src/ValidatedNumerics.jl

```diff
@@ -55,6 +57,12 @@ end
 export
     IntervalBox, @intervalbox
 
+## Decorations
+export
+    @decorated,
+    interval, decoration, DecoratedInterval,
```

Comment:
Maybe `interval` is not the best name (my fault, I know). `interval_part`? Does this need to be exported?